usaspending_api/api_contracts/README.md

```diff
@@ -7,6 +7,23 @@ All new API endpoint documentation must be in markdown that conforms to the [API
 ## For Developers
 See [Tools for Testing and Mocking](tools-for-contracts.md)
 
+To spin up a mock server, see the following instructions below:
```

Comment:
Would this make more sense in `tools-for-contracts.md`?

Reply:
Thanks @tony-sappe. Changes submitted.